packages/sui-segment-wrapper/src/segmentWrapper.js

```diff
@@ -39,11 +39,15 @@ const getTrackIntegrations = async ({gdprPrivacyValue, event}) => {
   let sessionId
 
   if (isGdprAccepted) {
-    ;[marketingCloudVisitorId, clientId, sessionId] = await Promise.all([
-      getAdobeMCVisitorID(),
-      getGoogleClientID(),
-      getGoogleSessionID()
-    ])
+    try {
```

Comment:
Allows event track to be sent even if there's some error on retrieving these integrations.